paddle/framework/eigen.h

```diff
@@ -65,13 +65,15 @@ template <typename T, int MajorType = Eigen::RowMajor,
           typename IndexType = Eigen::DenseIndex>
 struct EigenVector : public EigenTensor<T, 1, MajorType, IndexType> {
   // Flatten is to reshape a Tensor into a one dimension EigenVector
```

Comment:
Maybe we can type alias a `Parent` inside `EigenVector` to make code below cleaner.

```
using Parent = EigenTensor<T, 1, MajorType, IndexType>;
Parent::Type Flatten(...) {
  return Parent::Type(...)
}
```